packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -174,4 +176,53 @@ describe('SdkClient', async function () {
       sinon.assert.calledOnce(convertGasPriceToTinyBarsStub);
     });
   });
+
+  describe('HAPIService', async () => {
+    const OPERATOR_KEY_ED25519 = {
+      DER: '302e020100300506032b65700422042091132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137',
+      HEX_ED25519: '0x91132178e72057a1d7528025956fe39b0b847f200ab59b2fdd367017f3087137',
+    };
+
+    const OPERATOR_KEY_ECDSA = {
+      DER: '3030020100300706052b8104000a0422042008e926c84220295b5db5df25be107ce905b41e237ac748dd04d479c23dcdf2d5',
+      HEX_ECDSA: '0x08e926c84220295b5db5df25be107ce905b41e237ac748dd04d479c23dcdf2d5',
+    };
+
+    it('Initialize the privateKey for default which is DER', async () => {
```

Comment:
Since the default is now throwing error, we should update this test case. 
I think we need:
- a test case that process.env.OPERATOR_KEY_FORMAT is not configured, i.e. undefined
- a test case that process.env.OPERATOR_KEY_FORMAT is null
- a test case that process.env.OPERATOR_KEY_FORMAT is set to arbitrary value and expect an error to be thrown

Reply:
There is already a test to cover an arbitrary value that expects an error.  It's 'It should throw an Error when an unexpected string is set'.  The other suggestions are done.